src/panels/lovelace/entity-rows/hui-input-select-entity-row.ts

```diff
@@ -16,13 +16,16 @@ import { hasConfigOrEntityChanged } from "../common/has-changed";
 import "../components/hui-generic-entity-row";
 import { createEntityNotFoundWarning } from "../components/hui-warning";
 import { LovelaceRow } from "./types";
+import type { HaSelect } from "../../../components/ha-select";
 
 @customElement("hui-input-select-entity-row")
 class HuiInputSelectEntityRow extends LitElement implements LovelaceRow {
   @property({ attribute: false }) public hass?: HomeAssistant;
 
   @state() private _config?: EntitiesCardEntityConfig;
 
+  @query("ha-select", true) private _haSelect!: HaSelect;
```

Comment:
We can't cache this, as we will not always render the select, for example when there is an error or no config/hass yet

Reply:
If I understand correctly, all you're asking is to remove the cache option for @query, which I have just uploaded.

Reply:
Think it's ok for the state-summary card? I'm guessing that one will always be valid, but I don't know all the corner cases there.